src/cli/README.md

```diff
@@ -1581,6 +1582,41 @@ Set the link quality on the link to a given extended address.
 Done
 ```
 
+### locate \<anycastaddr\>
+
+Locate the closest destination of an anycast address (i.e., find the destination's mesh local EID and RLOC16).
+
+`OPENTHREAD_CONFIG_TMF_ANYCAST_LOCATOR_ENABLE` is required.
+
+The closest destination is determined based on the the current routing table and path costs within the Thread mesh.
+
+Locate the leader using its anycast address:
+
+```bash
+> locate fdde:ad00:beef:0:0:ff:fe00:fc00
+Locate result fdde:ad00:beef:0:d9d3:9000:16b:d03b RLOC16: 0xc800
+Done
+```
+
+Locate the closest destination of a service anycast address:
+
+```bash
+
+> srp server enable
+Done
+
+> netdata show
+Prefixes:
+Routes:
+Services:
+44970 5d c002 s c800
+44970 5d c002 s cc00
+Done
+
+> locate fdde:ad00:beef:0:0:ff:fe00:fc10
+Locate result fdde:ad00:beef:0:a477:dc98:a4e4:71ea RLOC16: 0xcc00
```

Comment:
```suggestion
fdde:ad00:beef:0:a477:dc98:a4e4:71ea RLOC16: 0xcc00
```
nit: remove redundant output to make it easier to parse.

Reply:
The `locate` command is async and the response can come back later (specially in cases of failure it can be much later). So added the extra `Locate result` to make it clear this is the outcome of a previous `locate` command.

Reply:
SGTM.